src/components/mobile-nav.tsx

```diff
@@ -11,27 +11,35 @@ import {
   IconButtonProps,
   useBreakpointValue,
   useColorModeValue,
+  useId,
   useUpdateEffect,
 } from '@chakra-ui/react'
 import { AnimatePresence, motion, useElementScroll } from 'framer-motion'
 import useRouteChanged from 'hooks/use-route-changed'
 import { getRoutes } from 'layouts/mdx'
 import NextLink from 'next/link'
 import { useRouter } from 'next/router'
-import * as React from 'react'
 import { AiOutlineMenu } from 'react-icons/ai'
 import { RemoveScroll } from 'react-remove-scroll'
 import Logo from './logo'
-import { SidebarContent } from './sidebar/sidebar'
+import {
+  isMainNavLinkActive,
+  mainNavLinks,
+  SidebarContent,
+} from './sidebar/sidebar'
 import SponsorButton from './sponsor-button'
-import { t } from 'utils/i18n'
+import { forwardRef, ReactNode, Ref, useEffect, useRef, useState } from 'react'
```

Comment:
We should move this import to the top to separate it from the local imports.

Reply:
We could add a eslint rule for this in a new PR, to make sure it is marked in the future.